components/shared/Button/button.tsx

```diff
@@ -29,26 +29,25 @@ const Button = React.forwardRef<HTMLElement, ButtonsProps>(
   ) => {
     const styles: Record<ButtonsProps["variant"], string> = {
       primary: `bg-light-orange-9 text-light-orange-2 border-light-orange-9 hover:bg-light-orange-10 ${
-        disabled ? "bg-light-orange-7 hover:bg-light-orange-7 pointer-events-none" : ""
```

Comment:
Pointer events being set to none is not necessary since you can still interact via the keyboard. As well, it prevents the mouse cursor from changing to not allowed.